packages/react/toggle-group/src/ToggleGroup.tsx

```diff
@@ -227,48 +250,41 @@ type ToggleGroupItemPrimitive = Polymorphic.ForwardRefComponent<
 >;
 
 const ToggleGroupItem = React.forwardRef((props, forwardedRef) => {
+  const valueContext = useToggleGroupValueContext(ITEM_NAME);
   const context = useToggleGroupContext(ITEM_NAME);
-  return context.rovingFocus ? (
-    <ToggleGroupRovingFocusItem {...props} ref={forwardedRef} />
-  ) : (
-    <ToggleGroupItemImpl {...props} ref={forwardedRef} />
-  );
+  const pressed = valueContext.value.includes(props.value);
+  const disabled = context.disabled || props.disabled;
```

Comment:
I changed this to calculate `pressed` and `disabled` in one place, and pass them down to both implementation.